plugins/mongo_db/mongo_db_state.cpp

```diff
@@ -239,7 +243,30 @@ namespace mongo_db {
     }
 
     auto state_writer::operator()(const transfer_operation& op) -> result_type {
-        
+        auto doc = create_document("transfer", "", "");
+        auto& body = doc.doc;
+
+        format_value(body, "from", op.from);
+        format_value(body, "to", op.to);
+        format_value(body, "amount", op.amount);
+        format_value(body, "memo", op.memo);
+
+        std::vector<std::string> part;
+        auto path = op.memo;
+        boost::split(part, path, boost::is_any_of("/"));
+        if (!part[0].empty() && part[0][0] == '@') {
```

Comment:
Add checks on size of parts >= 2. Because on the line 259 you try to get value from part[1], which can be absent.